site/app/controllers/EmailStatusController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace app\controllers;
+
+use app\libraries\Core;
+use app\libraries\response\MultiResponse;
+use app\libraries\response\WebResponse;
+use app\libraries\response\JsonResponse;
+use app\libraries\response\RedirectResponse;
+use app\models\email\EmailStatusPage;
+use Symfony\Component\Routing\Annotation\Route;
+use app\libraries\routers\AccessControl;
+
+class EmailStatusController extends AbstractController {
+    public function __construct(Core $core) {
+        parent::__construct($core);
+    }
+
+    /**
+     * @Route("/courses/{_semester}/{_course}/email_status")
+     * @return MultiResponse
+     */
+    public function getEmailStatusPage() {
+        $course = $this->core->getConfig()->getCourse();
+        $semester = $this->core->getConfig()->getSemester();
+        $result = $this->core->getQueries()->getEmailStatusWithCourse($course, $semester);
+        return MultiResponse::webOnlyResponse(
+            new WebResponse(
+                "app\\views\\email\EmailStatusView",
```

Comment:
The response allows passing in a "class string" by using the `::class` keyword on the class (e.g. `\app\views\email\EmailStatusView::class` see https://www.php.net/manual/en/language.oop5.basic.php#language.oop5.basic.class.class for more details on it). It's better to use a class string where possible as it allows for stronger static checking of the codebase. An example of this would be that if we were to delete the view file referenced here, in using a plain string phpstan would raise no error here, where if we did `\app\views\email\EmailStatusView::class`, then phpstan would complain.